### PR TITLE
[Learn] Add Info component on Learn homepage.

### LINKED ISSUE #1867: [Learn] Add Course, Chapter count component on Learn homepage
**Current State:**
The learn homepage i.e the page at layer5.io/learn-ng misses a component that should highlight the Courses, chapters, service-meshes count.

**Desired State:**
The required component needs to be added which highlights the courses, chapters, service-meshes count. These values should be fetched dynamically using graphql.
Reference component: [SiteStat](https://github.com/chromaui/learnstorybook.com/blob/master/src/components/screens/IndexScreen/SiteStats.js)

**Expected Design**
Screenshot from 2021-05-18 23-59-32

Link to [figma design](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=6090%3A13552)

---
**Contributor Resources**
The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- Check the Learn Layer5 site [Design Spec](https://docs.google.com/document/d/1tQ9tLbetBVKP7z76FR19vsxbAfoFHZzZYxCt7lJpAhU/edit) to know more about the site structure and development plans.
- Checkout the Learn Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/?node-id=4853%3A0).

### PR DESCRIPTION
**Pull request by Gautam-Arora24**
Signed-off-by: Gautam Arora <gautamarora6248@gmail.com>

**Description**
This PR intends to add a component on Learn homepage which will display the courses and chapter count.

**Caveats**
The current data is hard coded and is not fetched dynamically using GraphQL.

**Some Screenshots**
Screenshot 2021-05-21 at 1 19 37 AM
Screenshot 2021-05-21 at 1 20 46 AM

This PR fixes #1867 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
You still didn't change the typo :p

**Comment by Nikhil-Ladha** [comment]
Ohh, I see it's updated in the PR but the SS is old. No problem :)

File: src/sections/Learn-Layer5/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from "react";
 import { LearnLayer5GridWrapper , CardComponentWrapper} from "./learnlayer5.style";
 import { graphql, useStaticQuery } from "gatsby";
 import CardComponent from "../../components/Learn-Components/Card-Component";
+import Info from "./Info/index";
```
Comment (Nikhil-Ladha) [inline src/sections/Learn-Layer5/index.js:5]: Let's rename the component to `LearnInfo`

File: src/sections/Learn-Layer5/Info/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import CardWrapper from "./info.style";
+
+function Info() {
+  return (
+    <CardWrapper>
+      <div className="row">
```
Comment (Nikhil-Ladha) [inline src/sections/Learn-Layer5/Info/index.js:7]: You can import the `Row` and `Column` components from the `layout` file and use them here. Also, the `Container` component.
Component link: https://github.com/layer5io/layer5/blob/master/src/components/layout.js
For reference, you can check this [section](https://github.com/layer5io/layer5/blob/master/src/sections/Blog/Blog-grid/index.js) for usage.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 33566ae26842c806c152ada858fa70c0be2ee323 at: https://60a80452e2504d0f86af458a--layer5ng.netlify.app

File: src/sections/Learn-Layer5/index.js (hunk 1 above)
Comment (Gautam-Arora24) [reply src/sections/Learn-Layer5/index.js:5]: Yep, that sounds a reasonable name :)

File: src/sections/Learn-Layer5/Info/index.js (hunk 2 above)
Comment (Nikhil-Ladha) [reply src/sections/Learn-Layer5/Info/index.js:7]: Just an update let's not use the `Container` in the component directly. Let it be dependent upon the section where it is used, so we get flexibility.

Comment (Gautam-Arora24) [reply src/sections/Learn-Layer5/Info/index.js:7]: You mean to enclose the `LearnInfo` component inside its caller, right?

Comment (Gautam-Arora24) [reply src/sections/Learn-Layer5/Info/index.js:7]: So you mean to add the `Container` component in the file where I have called `LearnInfo`?

Comment (Nikhil-Ladha) [reply src/sections/Learn-Layer5/Info/index.js:7]: For the time being you can leave that. Just use the Row, and Column component in the LearnInfo component.

Comment (Gautam-Arora24) [reply src/sections/Learn-Layer5/Info/index.js:7]: Without using the `Container` in the LearnInfo the columns are not well organized. They are shifted towards the left. Does that make sense to you? 
Screenshot 2021-05-22 at 1 22 32 AM

File: src/sections/Learn-Layer5/Info/info.style.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+
+const CardWrapper = styled.div`
```
Comment (Nikhil-Ladha) [inline src/sections/Learn-Layer5/Info/info.style.js:3]: You can simply align them by adding a CSS here just before the class's CSS.

File: src/sections/Learn-Layer5/Info/index.js (hunk 2 above)
Comment (Nikhil-Ladha) [reply src/sections/Learn-Layer5/Info/index.js:7]: Check the comment below. It can be aligned by aligning the parent div.

**Comment by Nikhil-Ladha** [comment]
Once you make the above changes, I will merge the PR.
I will create a separate PR for the graphql changes, as I want some other changes to be made to the content structure as well.

**Comment by Gautam-Arora24** [comment]
Sure! Nikhil, I will make the changes and send the PR at end of the day.

**Comment by Nikhil-Ladha** [comment]
> Sure! Nikhil, I will make the changes and send the PR by the end of the day. Is that fine?

Not a new PR, but just add a commit to this one.

**Comment by Gautam-Arora24** [comment]
Done the changes requested ->
• Changed the name of the component from `Info` to `LearnInfo`.
• Used the `Column` and `Row` component from layouts.

**Comment by l5io** [comment]
🚀 Preview for commit 95db14115cae0cbec777c5cdbc021eb3ed8b70ad at: https://60a94dc0c56e6eac18672043--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
Looks good. Thank you @Gautam-Arora24 🚀

**Pull request merged**